salt/states/flatpak.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+'''
+Management of flatpak packages
+==============================
+Allows the installation and uninstallation of flatpak packages.
+
+.. versionadded:: Neon
+'''
+from __future__ import absolute_import, print_function, unicode_literals
+
+import salt.utils.path
+
+__virtualname__ = 'flatpak'
+
+
+def __virtual__():
+    if salt.utils.path.which('flatpak'):
+        return __virtualname__
+
+    return (False, 'The flatpak state module cannot be loaded: the "flatpak" binary is not in the path.')
+
+
+def installed(location, name):
+    '''
+    Ensure that the named package is installed
+    location
+        The location or remote to install the flatpak from.
+    name
+        The name of the package or runtime
+    '''
```

Comment:
Need Google Style Python Docstrings
Need Returns
Need Example of how a state file should look that uses this function:
```
Example:

.. code-block:: bash
    install_package:
      flatpak.installed:
        - location: something
        - name: some name
```
I don't know what values this is expecting

Reply:
This needs to be applied to all functions in this state module